apps/mull-ui/src/app/pages/create-event/create-event.scss

```diff
@@ -45,6 +45,63 @@
   margin-bottom: 15px;
 }
 
+.textarea-create-event-container {
```

Comment:
💡 General suggestions for the changes made in this file
* use variables for colours in case they are useful elsewhere
* use rem over px